### ISSUE TITLE
fortiotel variant isn't showing in footer (anymore)

### ISSUE THREAD

**Issue by ldemailly** [fortio/fortiotel#149]
docker run -p 8080:8080  fortio/fortiotel:1.60.0 server

shows at the bottom of http://localhost:8080/fortio/

Fortio 1.60.0 h1:0vlVg3/po/i1iXfWtBxlL/REOMyx7rCMuZIuzYAEW4k= go1.20.8 arm64 linux

while

 docker run -p 8080:8080  fortio/fortiotel:1.12.0 server

shows

Fortio 1.58.0 h1:Bpg8EcF454EK8ZT1kDrFqfu+OErttZICdWrnBeJWHI8= go1.20.7 arm64 linux (in fortio.org/fortiotel 1.12.0)

most likely a bug in https://github.com/fortio/version when the versions aligns

**Comment by ldemailly** [comment]
Non otel shows

Fortio 1.60.0 h1:0vlVg3/po/i1iXfWtBxlL/REOMyx7rCMuZIuzYAEW4k= go1.20.8 arm64 linux

```
$ docker run -p 8080:8080  fortio/fortiotel:1.60.0 buildinfo
1.60.0 h1:j3gjVbVYh8sLgI85PgtVfsrckT4V3NRpGEP3EpG0kdo= go1.20.8 arm64 linux
go	go1.20.8
path	fortio.org/fortiotel
mod	fortio.org/fortiotel	v1.60.0	h1:j3gjVbVYh8sLgI85PgtVfsrckT4V3NRpGEP3EpG0kdo=
dep	fortio.org/cli	v1.4.2	h1:pCYVPk5FpQuJtD31f9CG8sdgoTOWdv9Gls8As/MLJhU=
dep	fortio.org/dflag	v1.5.3	h1:+pCqZBCz95PYTxhcmenjrkVORWIRB1Je4eO/So574Bc=
dep	fortio.org/fortio	v1.60.0	h1:0vlVg3/po/i1iXfWtBxlL/REOMyx7rCMuZIuzYAEW4k=
dep	fortio.org/log	v1.11.0	h1:w7ueGPGbXz0A3+ApMz/5Q9gwEMrwSo/ohTlLo2Um6dU=
dep	fortio.org/scli	v1.11.0	h1:MI0kTivwCIF2L6eJJFUnTipd5d1KtCM1oVMB6Dcf6pY=
dep	fortio.org/sets	v1.0.3	h1:HzewdGjH69YmyW06yzplL35lGr+X4OcqQt0qS6jbaO4=
dep	fortio.org/version	v1.0.2	h1:8NwxdX58aoeKx7T5xAPO0xlUu1Hpk42nRz5s6e6eKZ0=
dep	github.com/cenkalti/backoff/v4	v4.2.1	h1:y4OZtCnogmCPw98Zjyt5a6+QwPLGkiQsYW5oUqylYbM=
dep	github.com/felixge/httpsnoop	v1.0.3	h1:s/nj+GCswXYzN5v2DpNMuMQYe+0DDwt5WVCU6CWBdXk=
dep	github.com/fsnotify/fsnotify	v1.6.0	h1:n+5WquG0fcWoWp6xPWfHdbskMCQaFnG6PfBrh1Ky4HY=
dep	github.com/go-logr/logr	v1.2.4	h1:g01GSCwiDw2xSZfjJ2/T9M+S6pFdcNtFYsp+Y43HYDQ=
dep	github.com/go-logr/stdr	v1.2.2	h1:hSWxHoqTgW2S2qGc0LTAI563KZ5YKYRhT3MFKZMbjag=
dep	github.com/golang/protobuf	v1.5.3	h1:KhyjKVUg7Usr/dYsdSqoFveMYd5ko72D+zANwlG1mmg=
dep	github.com/google/uuid	v1.3.1	h1:KjJaJ9iWZ3jOFZIf1Lqf4laDRCasjl0BCmnEGxkdLb4=
dep	github.com/grpc-ecosystem/grpc-gateway/v2	v2.16.0	h1:YBftPWNWd4WwGqtY2yeZL2ef8rHAxPBD8KFhJpmcqms=
dep	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace	v0.44.0	h1:ewRgsETI7b5nPCK3FqKdY9mFR/9ZwtexwC26//Srjn0=
dep	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp	v0.44.0	h1:KfYpVmrjI7JuToy5k8XV3nkapjWx48k4E4JOtVstzQI=
dep	go.opentelemetry.io/contrib/propagators/b3	v1.19.0	h1:ulz44cpm6V5oAeg5Aw9HyqGFMS6XM7untlMEhD7YzzA=
dep	go.opentelemetry.io/otel	v1.18.0	h1:TgVozPGZ01nHyDZxK5WGPFB9QexeTMXEH7+tIClWfzs=
dep	go.opentelemetry.io/otel/exporters/otlp/otlptrace	v1.18.0	h1:IAtl+7gua134xcV3NieDhJHjjOVeJhXAnYf/0hswjUY=
dep	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc	v1.18.0	h1:yE32ay7mJG2leczfREEhoW3VfSZIvHaB+gvVo1o8DQ8=
dep	go.opentelemetry.io/otel/metric	v1.18.0	h1:JwVzw94UYmbx3ej++CwLUQZxEODDj/pOuTCvzhtRrSQ=
dep	go.opentelemetry.io/otel/sdk	v1.18.0	h1:e3bAB0wB3MljH38sHzpV/qWrOTCFrdZF2ct9F8rBkcY=
dep	go.opentelemetry.io/otel/trace	v1.18.0	h1:NY+czwbHbmndxojTEKiSMHkG2ClNH2PwmcHrdo0JY10=
dep	go.opentelemetry.io/proto/otlp	v1.0.0	h1:T0TX0tmXU8a3CbNXzEKGeU5mIVOdf0oykP+u2lIVU/I=
dep	golang.org/x/exp	v0.0.0-20230905200255-921286631fa9	h1:GoHiUyI/Tp2nVkLI2mCxVkOjsbSXD66ic0XW0js0R9g=
dep	golang.org/x/net	v0.15.0	h1:ugBLEUaxABaB5AJqW9enI0ACdci2RUd4eP51NTBvuJ8=
dep	golang.org/x/sys	v0.12.0	h1:CM0HF96J0hcLAwsHPJZjfdNzs0gftsLfgKt57wWHJ0o=
dep	golang.org/x/text	v0.13.0	h1:ablQoSUd0tRdKxZewP80B+BaqeKJuVhuRxj/dkrun3k=
dep	google.golang.org/genproto/googleapis/api	v0.0.0-20230711160842-782d3b101e98	h1:FmF5cCW94Ij59cfpoLiwTgodWmm60eEV0CjlsVg2fuw=
dep	google.golang.org/genproto/googleapis/rpc	v0.0.0-20230913181813-007df8e322eb	h1:Isk1sSH7bovx8Rti2wZK0UZF6oraBDK74uoyLEEVFN0=
dep	google.golang.org/grpc	v1.58.0	h1:32JY8YpPMSR45K+c3o6b8VL73V+rR8k+DeMIr4vRH8o=
dep	google.golang.org/protobuf	v1.31.0	h1:g0LDEJHgrBl9N9r17Ru3sqWhkIx2NB67okBHPwC7hs8=
build	-buildmode=exe
build	-compiler=gc
build	-ldflags="-s -w -X main.version=1.60.0 -X main.commit=805daa0faf392d05773e8859f51d220f92d31ecd -X main.date=2023-09-14T18:13:10Z -X main.builtBy=goreleaser"
build	CGO_ENABLED=0
build	GOARCH=arm64
build	GOOS=linux
```

**Comment by ldemailly** [comment]
because of

https://github.com/fortio/version/blob/v1.0.2/version.go#L76

**Comment by ldemailly** [comment]
Will be fixed once fortio/version 1.0.3 propagates to fortio (so 1.60.1)
